src/libs/antares/series/include/antares/series/series.h

```diff
@@ -35,10 +35,17 @@ namespace Antares::Data
 class TimeSeries
 {
 public:
-    using numbers = Matrix<uint32_t>;
+    struct Numbers : public Matrix<uint32_t> // TODO[FOM] remove inheritance, it was added mostly to avoid breaking existing code
```

Comment:
**TimeSeries** should know nothing about TS numbers, I think class **TimeSeries** should be left alone.

On the other hand, class **Numbers** (to be renamed into **TSnumbers**) should be the one who knows about **TimeSeries**, because it checks some **TimeSeries** nmuber of columns consistency.

So, to me : 
- This class **Numbers** should be renamed into **TSnumbers**, and moved out from class **TimeSeries** (probably in a separate source file).
- We should reverse the registration process : make **Numbers** register **TimeSeries**, and not (as it is done now) make **TimeSeries** allow **Numbers** to register them (which is more complicated). 

For example, `DataSeriesHydro::DataSeriesHydro()` should make `timeseriesNumbers` register `ror`, `storage `and `mingen`.

Reply:
Yes, will do

Reply:
DONE